myapp/spec/factories/task.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+    factory :task, class: Task do
+        sequence(:title) { |n| "test_title#{n}" }
+        description { 'spec_description' }
+    end
+end
```

Comment:
I fix it thanks!
I run rubocop, but it could not add a blank line at the end of the file.
I just `gem install fablicop` and run `rubocop -a`.
Do I need any other settings...?

Reply:
replied on slack